extensions/typescript-language-features/src/tsServer/spawner.ts

```diff
@@ -268,6 +268,7 @@ export class TypeScriptServerSpawner {
 		args.push('--locale', TypeScriptServerSpawner.getTsLocale(configuration));
 
 		args.push('--noGetErrOnBackgroundUpdate');
+		args.push('--canUseWatchEvents'); // TODO check ts version
```

Comment:
I think this is just

```suggestion
		if (apiVersion.gte(API.v520)) {
			args.push('--canUseWatchEvents');
		}
```

Since versions made using `fromVersionString` throw away pre-releases tags.